_posts/2016-05-30-pitest-with-java.md

```diff
@@ -0,0 +1,263 @@
+---
+layout: post
+title:  "Test your tests with pitest"
+date:   2016-05-30 08:00:00 +0200
+author: "Tomasz Banaś"
+tags: java tests
+---
+
+There is no doubt that unit tests are necessary to keep good quality of code. People used to check line coverage to determine if they have good tests, but have you ever wonder how to test your tests and check if they really tests something?
+In this post I'll show show you why line coverage is a bad metric and how to use mutation testing to test unit tests.
```

Comment:
"... how to use mutation testing in Java to test unit tests."
I would mention we are talking about mutation testing in Java. More keywords for google and it's clear for readers.